practice-two/src/pages/Analytics/Analytics.jsx

```diff
@@ -29,6 +29,12 @@ const Analytics = () => {
   const [selectedCustomer, setSelectedCustomer] = useState(null);
   const [isShowProfileInfo, setIsShowProfileInfo] = useState(false);
   const [isShowContextMenu, setIsShowContextMenu] = useState(false);
+  const [isShowForm, setIsShowForm] = useState(false);
+
+  // Handle open/close form
+  const handleShowForm = useCallback(() => {
+    setIsShowForm(!isShowForm);
+  }, [isShowForm]);
 
   // Event handler for clicking a customer
   const handleShowProfileInfo = (customer) => {
```

Comment:
check re-render for function

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/1d828d62-1c45-4788-aaf0-81e9802ba602)